### PR TITLE
Add homebrew installation instructions for cp-remote

### PR DESCRIPTION
**Pull request by andytson-inviqa**

### CODE REVIEW

File: content/remote-development/getting-started.md [hunk 1]
```diff
@@ -28,6 +28,13 @@ If the GitHub repository is not the origin of your checked out project then you
 
 ### OSX (64-bit):
 
+If you use [Homebrew](https://brew.sh/), you can install cp-remote via:
```
Comment (pswaine) [inline content/remote-development/getting-started.md]: Please backtick `cp-remote` for consistency

Comment (andytson-inviqa) [inline content/remote-development/getting-started.md]: I don't think cp-remote should be backticked unless it's part of a command

Comment (pswaine) [inline content/remote-development/getting-started.md]: Is that a personal preference or a wider standard? (genuine question) The difficulty is that I'm trying to maintain consistency across the document and the rest of the documents. Rightly or wrongly, backticks have been used for much more than just command expressions - really they are being used to highlight anything that looks like a command, application name, variable or filename. This is more than I would have done if I'd written everything from scratch, but presentationally it is useful as it distinguishes technical terms from regular content.

**Comment by sroze** [comment]
That makes me think... how do we release upgrades for this homebrew version? Where is the "manifest" ?

File: content/remote-development/getting-started.md [hunk 2]
```diff
@@ -28,6 +28,13 @@ If the GitHub repository is not the origin of your checked out project then you
...
+
+```
+brew install inviqa/tools/cp-remote
```
Comment (sroze) [inline content/remote-development/getting-started.md]: I think it'd be better to do `continuouspipe/cp-remote` tbh

Comment (andytson-inviqa) [inline content/remote-development/getting-started.md]: can do `continuouspipe/tools/cp-remote, but you'd need to create a `continuouspipe/homebrew-tools` repo

Comment (andytson-inviqa) [inline content/remote-development/getting-started.md]: in the longer run, getting cp-remote into homebrew-core might be better (tho that might slow down the release process relying on homebrew devs to merge updates)

**Comment by andytson-inviqa** [comment]
what do you mean about "manefest"?

**Comment by andytson-inviqa** [comment]
release upgrades would be through updating the version number of https://github.com/inviqa/homebrew-tools/blob/master/Formula/cp-remote.rb#L4, which I've said to @alessandrozucca

**Comment by tonypiper** [comment]
how can we get this one over the line, @pswaine

**Pull request merged**